README.md

```diff
@@ -47,6 +47,19 @@ DogeHouse is open to contributions, but I recommend creating an issue or replyin
 
 Please read [CONTRIBUTING.md](https://github.com/benawad/dogehouse/blob/staging/CONTRIBUTING.md) for details on this project.
 
+## Dogehouse Desktop
+
+A desktop app built with electron is available for Windows and Mac.
+
+|      Downloads       |                       |
+| :------------------- | :-------------------: |
+| [Dogehouse-Setup-1.0.16.exe](https://github.com/benawad/dogehouse/releases/download/v1.0.16/DogeHouse-Setup-1.0.16.exe)       |      Windows      |
+| [Dogehouse-1.0.16.dmg](https://github.com/benawad/dogehouse/releases/download/v1.0.16/DogeHouse-1.0.16.dmg) |     MacOS      |
```

Comment:
dont hard code in the links. if a new release will be published these links will be outdated

Reply:
@isaboo you can replace the `v1.0.16` with `latest` to keep the links up to date

Reply:
What about the v1.0.16 in the file name
(e.g latest/doghouse-setup-**_1.0.16_**.exe

Reply:
I'd probably just remove those links and stick with the single link for [latest](https://github.com/benawad/dogehouse/releases/latest). People will know which is for their OS.